docs/writing-docs/docs-page.md

```diff
@@ -1,205 +1,189 @@
 
 <!-- prettier-ignore-start -->
 
 <CodeSnippets
   paths={[
-    'common/button-story-remix-docspage.js.mdx',
-    'common/button-story-remix-docspage.ts.mdx',
+    'common/storybook-auto-docs-starter-example.mdx.mdx',
   ]}
 />
 
 <!-- prettier-ignore-end -->
 
-In addition, you can interleave your own components to customize the auto-generated contents of the page or pass in different options to the blocks to customize their appearance.
-
-## Story file names
+### Working with multiple components
 
-Unless you use a custom [Webpack configuration](../builders/webpack.md#extending-storybooks-webpack-config), all of your story files should have the suffix `*.stories.@(j|t)sx?`. For example, "Badge.stories.js" or "Badge.stories.tsx". This tells Storybook and its docs preset to display the docs based on the file contents.
+If you need to document multiple components in a single documentation page, you can reference them directly inside your MDX file. Internally, Storybook looks for the story metadata and composes it alongside your existing documentation. For example:
 
-## Inline stories vs. iframe stories
+<!-- prettier-ignore-start -->
 
-DocsPage displays all the stories of a component on one page. You have the option of rendering those stories inline or in an iframe.
+<CodeSnippets
+  paths={[
+    'common/storybook-auto-docs-mdx-file.mdx.mdx',
+  ]}
+/>
 
-The iframe creates a clean separation between your code and Storybook’s UI, which is useful if your stories are rendering correctly in the Canvas but not on the docs page, for instance with fixed positioned components like modals.
+<!-- prettier-ignore-end -->
 
-But using an iframe has disadvantages. For example, you have to set the height of iframe stories explicitly, or you’ll see a scroll bar. Having more than a few iframe stories on a page can lead to performance issues. And certain dev tools might not work right.
+## Advanced configuration
 
-Therefore, we recommend inline rendering where possible. It's the default mode for all the frameworks in which [we support it](../api/frameworks-feature-support.md). The one exception is Angular, where it's opt-in.
+### Customize the Docs Container
 
-To toggle the between the two settings, set `docs.inlineStories` in `.storybook/preview.js`. Like most [parameters](../writing-stories/parameters.md), you can also toggle at the component or story level:
+The Docs Container is the component that wraps up the documentation page. It's responsible for rendering the documentation page in Storybook's UI. You can customize it by creating your own component and updating your Storybook UI configuration file (i.e., `.storybook/preview.js`) to reference it.
```

Comment:
I think it's important to mention that we strongly recommend rendering the SB DocsContainer inside their custom one, because it provides context used by various Docs Blocks.